core/scripts/vrfv2/testnet/main.go

```diff
@@ -854,6 +855,112 @@ func main() {
 		resp, err := coordinator.GetSubscription(nil, *subID)
 		helpers.PanicErr(err)
 		fmt.Println("sub id", *subID, "balance:", resp.Balance)
+	case "coordinator-withdrawable-tokens":
+		withdrawableTokensCmd := flag.NewFlagSet("coordinator-withdrawable-tokens", flag.ExitOnError)
+		coordinator := withdrawableTokensCmd.String("coordinator-address", "", "coordinator address")
+		oracle := withdrawableTokensCmd.String("oracle-address", "", "oracle address")
+		helpers.ParseArgs(withdrawableTokensCmd, os.Args[2:], "coordinator-address", "oracle-address")
+
+		coordinatorAddress := common.HexToAddress(*coordinator)
+		oracleAddress := common.HexToAddress(*oracle)
+		abi, err := vrf_coordinator_v2.VRFCoordinatorV2MetaData.GetAbi()
+		helpers.PanicErr(err)
+
+		isWithdrawable := func(amount *big.Int) bool {
+			data, err := abi.Pack("oracleWithdraw", oracleAddress /* this can be any address */, amount)
+			helpers.PanicErr(err)
+
+			_, err = ec.CallContract(context.Background(), ethereum.CallMsg{
+				From: oracleAddress,
+				To:   &coordinatorAddress,
+				Data: data,
+			}, nil /* TODO */)
+			if err == nil {
+				return true
+			} else if strings.Contains(err.Error(), "execution reverted") {
+				return false
+			} else {
+				panic(err)
+			}
+		}
+
+		// Do a binary search to find the withdrawable amount
+
+		// top is the lowest non-withdrawable amount we have tested
+		top, _ := new(big.Int).SetString("10000000000000000000000", 10) // 10,000 LINK
```

Comment:
Wondering if this should be an env-var? We could conceivably have > 10,000 LINK on chains like e.g Ethereum.

Can also use `assets.Ether()` to make this more readable, at that point can just pass in the LINK value rather than Juels.

Reply:
Yeah good call, done (made it a param with a default value).